core/src/components/range/range.tsx

```diff
@@ -191,6 +191,18 @@ export class Range implements ComponentInterface {
    */
   @Event() ionBlur!: EventEmitter<void>;
 
+  /**
+   * Emitted when the user starts moving the range knob, whether through
+   * mouse drag, touch gesture, or keyboard interaction.
+   */
+  @Event() ionChangeStart!: EventEmitter<RangeChangeStartEventDetail>;
```

Comment:
Bikeshedding here, but thoughts on calling this `ionStart` and `ionEnd`? To me `ionChangeStart` implies that `ionChange` will fire after `ionChangeStart`. If you have a pointer on the knob, `ionChangeStart` will fire, but if you simply remove the point and do not move the knob, `ionChange` will not fire.